pkg/controller/controller_binding.go

```diff
@@ -221,6 +221,30 @@ func (c *controller) reconcileBinding(binding *v1alpha1.Binding) error {
 
 		response, err := brokerClient.Bind(request)
 		if err != nil {
+			if osb.IsHTTPError(err) {
+				httpErr := err.(osb.HTTPStatusCodeError)
+				s := fmt.Sprintf("Error creating Binding \"%s/%s\" for Instance \"%s/%s\" of ServiceClass %q at Broker %q with status code %d: ErrorMessage: %v, Description: %v",
+					binding.Name,
+					binding.Namespace,
+					instance.Namespace,
+					instance.Name,
+					serviceClass.Name,
+					brokerName,
+					httpErr.StatusCode,
+					*httpErr.ErrorMessage,
```

Comment:
Nevermind, it's nil safe but doesn't give us the behavior we want.  You'll need to make variables that hold the value to be printed ('<nil>' if not set), and print those.  Dereferencing these pointers is not safe.